00 Intro/00 BoilerPlate/README.md

```diff
@@ -34,32 +34,34 @@ npm init
 
 ![npm init](../../99 Readme Resources/02 Webpack/Demo00_npminit.png "Demo00_npminit.png")
 
-- Install webpack locally, as a development dependency (the reason to install it locally and not globally is to be easy to setup, e.g. can be launched on a clean machine without having to install anything globally but nodejs).
+- Install **webpack** locally, as a development dependency (the reason to install it locally and not globally is to be easy to setup, e.g. can be launched on a clean machine without having to install anything globally but nodejs).
 
 ````
 npm install webpack --save-dev
 ````
 
-- In order to launch webpack, modify the **package.json** file an add a property under the scripts object to run it. Now, our package.json file should looks something like:
+- In order to launch webpack, modify the **package.json** file an add the following property `"start": "webpack"` under the scripts object. It allows us to launch webpack from the command line through npm typing `npm start`.
+
```

Comment:
It would be a good idea to insert here the package.json snippet where you can find the start, or not sure if the snapshot below contains that.